src/installation/thirdparty.md

```diff
@@ -8,6 +8,42 @@ Those methods are not officially supported, if you encounter installation issues
 
 A [Powershell script](https://github.com/losslesspng/SetUpKomgaJava) to get up and running with Komga.
 
+## Scoop (Windows)
+
+Komga is available in [Scoop](https://github.com/ScoopInstaller/Scoop)'s [extras](https://github.com/ScoopInstaller/Extras) bucket.
+
+### Installation
+
+#### 1. (Skip if Scoop is installed) Install Scoop
+Open Powershell and run `iwr -useb get.scoop.sh | iex`. If you got an error, run `Set-ExecutionPolicy RemoteSigned -scope CurrentUser`.
+
+#### 1.1 (Optional) Install aria2 for multi-connection download
+Run `scoop install aria2`.
+
+#### 2. (Skip if JDK is installed) Add java bucket
+Run `scoop add bucket extras`.
+
+#### 3. (Skip if JDK is installed) Install JDK
+Run `scoop install java/temurin-lts-jdk`.
```

Comment:
I thought Scoop would handle dependencies in formulas, and install deps automatically. Isn't that the case?

Reply:
It could. Scoop have a way to handle dependencies but I set JDK as a `suggest` package intead of `dependency` in Scoop because there's a chance that user already install JDK outside Scoop.

Reply:
Oh. Isn't Scoop smart enough to detect local Java and not install if not needed?

Reply:
tbh I haven't tested that interaction. I know [it can have multiple java installation](https://github.com/ScoopInstaller/Scoop/wiki/Java#switching-javas) using scoop. Not sure if outside scoop installation is handled.  I'll check that later.

Reply:
I haven't tested it but I found some similiar case of detecting an app installed outside Scoop
https://github.com/ScoopInstaller/Extras/pull/8048
https://github.com/ScoopInstaller/Scoop/issues/1263
https://github.com/ScoopInstaller/Extras/issues/3200

I think they can't handle it yet. DBeaver[^1] and Selenium[^2] depends on java but in their app manifest, it's stated as `suggest`. I think we should state it as `suggest` too.

[^1]: https://github.com/ScoopInstaller/Extras/blob/master/bucket/dbeaver.json
[^2]: https://github.com/ScoopInstaller/Main/blob/master/bucket/selenium.json